docs/admin/extensible-admission-controllers.md

```diff
@@ -20,11 +21,169 @@ the following:
 * They are only configurable when the apiserver starts up.
 
-1.7 introduces two alpha features, *Initializers* and *External Admission
-Webhooks*, that address these limitations. These features allow admission
-controllers to be developed out-of-tree and configured at runtime.
+Two features, *Admission Webhooks* (beta in 1.9) and *Initializers* (alpha),
+address these limitations. They allow admission controllers to be developed
+out-of-tree and configured at runtime.
 
-This page describes how to use Initializers and External Admission Webhooks.
+This page describes how to use Admission Webhooks and Initializers.
+
+## Admission Webhooks
+
+### What are admission webhooks?
+
+Admission webhooks are HTTP callbacks that receive admission requests and do
+something with them. You can define two types of admission webhooks,
+[ValidatingAdmissionWebhooks](/docs/admin/admission-controllers.md#validatingadmissionwebhook-alpha-in-18-beta-in-19)
+and
+[MutatingAdmissionWebhooks](/docs/admin/admission-controllers.md#mutatingadmissionwebhook-beta-in-19).
+With `ValidatingAdmissionWebhooks`, you may reject requests to enforce custom
+admission policies. With `MutatingAdmissionWebhooks`, you may change requests to
+enforce custom defaults.
+
+### Experimenting with admission webhooks
+
+Admission webhooks are essentially part of the cluster control-plane. You should
+write and deploy them with great caution. Please read the [user
+guides](https://github.com/kubernetes/website/pull/6836/files)(WIP) for
+instructions if you intend to write/deploy production-grade admission webhooks.
+In the following, we describe how to quickly experiment with admission webhooks.
+
+### Prerequisites
+
+* Ensure that the Kubernetes cluster is at least as new as v1.9.
+
+* Ensure that MutatingAdmissionWebhook and ValidatingAdmissionWebhook
+  admission controllers are enabled.
+  [Here](/docs/admin/admission-controllers.md#is-there-a-recommended-set-of-admission-controllers-to-use)
+  is a recommended set of admission controllers to enable in general.
+
+* Ensure that the admissionregistration.k8s.io/v1beta1 API is enabled.
+
+### Write an admission webhook server
+
+Please refer to the implementation of the [admission webhook
+server](https://github.com/kubernetes/kubernetes/blob/v1.10.0-beta.1/test/images/webhook/main.go)
+that is validated in a Kubernetes e2e test. The webhook handles the
+`admissionReview` requests sent by the apiservers, and sends back its decision
+wrapped in `admissionResponse`.
+
+The example admission webhook server leaves the `ClientAuth` field
+[empty](https://github.com/kubernetes/kubernetes/blob/v1.10.0-beta.1/test/images/webhook/config.go#L48-L49),
+which defaults to `NoClientCert`. This means that the webhook server does not
+authenticate the identity of the clients, supposedly apiservers. If you need
+mutual TLS or other ways to authenticate the clients, see
+how to (authenticate apiservers)[#authenticate-apiservers].
```

Comment:
(authenticate apiservers)[#authenticate-apiservers]
should be
\[authenticate apiservers](#authenticate-apiservers)

Reply:
Done.